cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -57,6 +57,7 @@
   protected static final String PRODUCER_CLIENT_ID = "KafkaCruiseControlSampleStoreProducer";
   protected static final String CONSUMER_CLIENT_ID = "KafkaCruiseControlSampleStoreConsumer";
   private static final String DEFAULT_CLEANUP_POLICY = "delete";
+  private static final long MIN_SAMPLE_TOPIC_RETENTION_TIME_MS = 3600000L;
```

Comment:
Going forward, should we make this configurable?

Reply:
Not really, I am not even sure this should exist at all. I added it just because otherwise the quick start will have the historical data deleted too quickly.